DESCRIPTION

```diff
@@ -36,24 +36,30 @@ License: LGPL-3
 URL: https://mlr3tuning.mlr-org.com, https://github.com/mlr-org/mlr3tuning
 BugReports: https://github.com/mlr-org/mlr3tuning/issues
 Depends:
-    mlr3 (>= 0.14.0),
+    mlr3 (>= 0.14.0-9000),
     paradox (>= 0.7.0),
     R (>= 3.1.0)
 Imports:
-    bbotk (>= 0.5.2),
+    bbotk (>= 0.5.4.9000),
     checkmate (>= 2.0.0),
     data.table,
     lgr,
-    mlr3misc (>= 0.10.0),
+    mlr3misc (>= 0.11.0),
     R6
 Suggests:
     adagio,
     GenSA,
     irace,
+    mlr3learners (>= 0.5.4-9000),
     mlr3pipelines,
     nloptr,
     rpart,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
```

Comment:
Not really in this PR, but some comments on the callbacks in misc: 
https://github.com/mlr-org/mlr3misc/blob/74d4e4583ddcabbbcb07fc19375f21935446019c/R/Callback.R#L25 here we should show what the callback does when being executed in a function.
Also there should be a section talking about the order in which the callbacks are executed.

Reply:
Also I find it now a little weird that here https://github.com/mlr-org/mlr3misc/blob/74d4e4583ddcabbbcb07fc19375f21935446019c/R/Callback.R#L160 we do `callback$call("on_state", context)` instead of a simple `callback[["on_stage"]](context)`. The `call` method serves no purpose really except for adding an additional function call to the stack

Reply:
like with `as_learners` we can here: https://github.com/mlr-org/mlr3misc/blob/74d4e4583ddcabbbcb07fc19375f21935446019c/R/Callback.R#L147 implement a default method `as_callbacks.default = function(x, ...) list(as_callback(x, ...))`

Reply:
Is the `...` argument here: https://github.com/mlr-org/mlr3misc/blob/74d4e4583ddcabbbcb07fc19375f21935446019c/R/Callback.R#L198 actually being used somewhere? I guess we could do without it and rely and the mechanism that is implemented in `dictionary_sugar_get` that already allows to set fields and constructor arguments

Reply:
> Is the `...` argument here: https://github.com/mlr-org/mlr3misc/blob/74d4e4583ddcabbbcb07fc19375f21935446019c/R/Callback.R#L198 actually being used somewhere? I guess we could do without it and rely and the mechanism that is implemented in `dictionary_sugar_get` that already allows to set fields and constructor arguments

I think that got a bit mixed up. We should use the `$state` instead of fields. I will remove the option to create fields with `callback_tuning()`. The `... `passes to  `$state`.

Reply:
> Also I find it now a little weird that here https://github.com/mlr-org/mlr3misc/blob/74d4e4583ddcabbbcb07fc19375f21935446019c/R/Callback.R#L160 we do `callback$call("on_state", context)` instead of a simple `callback[["on_stage"]](context)`. The `call` method serves no purpose really except for adding an additional function call to the stack

Yes, we should change that in the next release.